all-dogs.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> All Dogs</title>
+ <script src="https://kit.fontawesome.com/2315930367.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+
+  <link rel="stylesheet" type= "text/css" href= "./styles/styles.css">
+  <script src="./script.js"></script>
+</head>
+<body>
+  <header>
+
+    <div class= "checkout">
+      <i class="fas fa-shopping-cart"></i>
+      <button type="button" onclick="alert('adoptCart')">Checkout</button>
+  </div>
+<div class= "logo">
+    <img src="./images/adoptadog-logo.svg" alt="logo">
+    <h1>Seattle’s premiere dog adoption agency</h1>
+</div>
+  </header>
+  <nav>
+      <ul>
+        <li><a href="./index.html">Home/</a></li>
+        <li><a href="./all-dogs.html">Dogs/</a></li>
+        <li><a href="./blog.html">Blog/</a></li>
+      </ul>
+  </nav>
+  <article class= "main-article">
```

Comment:
Try to not put things outside of the main element like this. Your main wrapper should be a wrapper around most of your main content. For this site I'd recommend everything btween the header and the footer in the main tag. Then the dog content in one section and the blog preview in a sidebar.